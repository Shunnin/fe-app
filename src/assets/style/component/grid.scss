/*======================================

  Grid Component
    -> Descriptions

======================================*/

/*--------------------------------------
  Container Widths
--------------------------------------*/

// Set the container width, and override it for fixed navbars in media queries.
.container {
  @include container-fixed;

  @media (min-width: $screen-sm-min) {
    max-width: $container-sm;
  }

  @media (min-width: $screen-md-min) {
    max-width: $container-md;
  }

  @media (min-width: $screen-lg-min) {
    max-width: $container-lg;
  }
}

/*--------------------------------------
  Fluid Container
--------------------------------------*/

// Utilizes the mixin meant for fixed width containers, but without any defined
// width for fluid, full width layouts.
.container-fluid {
  @include container-fixed;
}
