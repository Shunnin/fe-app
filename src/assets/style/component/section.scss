$section-padding: $b-space-2x;

/* ----- Section Header ----- */
.section {
  padding: $b-space;

  /* ----- Section Title ----- */
  &__title {
    margin: $section-padding;
    font-weight: $b-font-weight-medium;
    font-size: $b-line-height-heading;
  }

  /* ----- Section Content ----- */
  &__content {
    @include clearfix();

    padding: $b-space-2x;
  }
}
