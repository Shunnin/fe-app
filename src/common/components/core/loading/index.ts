export { Loading } from './loading.commponent';
