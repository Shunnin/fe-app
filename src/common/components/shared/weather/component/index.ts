export { Temperature } from './temperature/temperature.component';
export { WeatherCard } from './weather-card/weather-card.component';
export { WeatherItem } from './weather-item/weather-item.component';
export * from './weather-icon';
