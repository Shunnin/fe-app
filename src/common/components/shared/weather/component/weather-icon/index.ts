export { WeatherIcon } from './weather-icon.component';
export * from './weather-icon.constant';
