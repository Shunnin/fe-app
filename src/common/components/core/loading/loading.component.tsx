import { FC } from 'react';

import './loading.style';

export const Loading: FC = () => {
  return (
    <div className="loader">
      <div className="sk-chase">
        <div className="sk-chase-dot"></div>
        <div className="sk-chase-dot"></div>
        <div className="sk-chase-dot"></div>
        <div className="sk-chase-dot"></div>
        <div className="sk-chase-dot"></div>
        <div className="sk-chase-dot"></div>
      </div>
    </div>
  );
};
