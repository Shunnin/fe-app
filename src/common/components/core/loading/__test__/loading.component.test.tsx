import { render } from '@testing-library/react';

import { Loading } from '../loading.component';

describe('<Loading />', () => {
  it('renders and matches the snapshot', () => {
    const { container } = render(<Loading />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
