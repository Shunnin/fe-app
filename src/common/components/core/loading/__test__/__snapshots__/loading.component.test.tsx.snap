// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders and matches the snapshot 1`] = `
<div
  class="loader"
>
  <div
    class="sk-chase"
  >
    <div
      class="sk-chase-dot"
    />
    <div
      class="sk-chase-dot"
    />
    <div
      class="sk-chase-dot"
    />
    <div
      class="sk-chase-dot"
    />
    <div
      class="sk-chase-dot"
    />
    <div
      class="sk-chase-dot"
    />
  </div>
</div>
`;
